src/plugins/lowMach.cpp

```diff
@@ -54,7 +54,13 @@ void lowMach::setup(nrs_t* nrs, dfloat gamma)
   platform->options.setArgs("LOWMACH", "TRUE"); 
 }
 
+// qtl = (-1/(rho**2.0*Cp))*(drho/dt)*(div[k*grad[T] ] + qvol)
+// alphav = -(1/rho)(drho/dT)  ! thermal expension coefficient
+// for general real gas/fluid 
+// qtl = (alphav/(rho*Cp))*(div[k*grad[T] ] + qvol)
+// for ideal gas, alphav =  1/T
 // qtl = 1/(rho*cp*T) * (div[k*grad[T] ] + qvol)
+
 void lowMach::qThermalIdealGasSingleComponent(dfloat time, occa::memory o_div)
```

Comment:
lowMach::qThermalIdealGasSingleComponent should just call lowMach::qThermalRealGasSingleComponent but pass o_NULL for o_expansionCoeff. The inside qThermalIdealGasSingleComponent you check if `o_expansionCoeff.ptr() == NULL` and set a flag. Idea is two have to interfaces but one implementation.

Reply:
ok, found a way to do this.